targets/targets.json

```diff
@@ -3840,7 +3841,9 @@
             "FLASH",
             "QSPI",
             "MPU",
-            "USBDEVICE"
+            "USBDEVICE",
+            "SDIO",
+            "SDIO_ASYNC"
```

Comment:
We need to add COMPONENT_SDIO AND DEVICE_SDIO AND DEVICE_SDIO_ASYNC ?

Reply:
Yes